app/src/main/java/com/moxielabs/androidarchitectureexamples/mvc/MvcFragment.kt

```diff
@@ -46,6 +47,10 @@ class MvcFragment : Fragment() {
         }
     }
 
+    fun Fruit.displayText(): String {
+        return "${this.taste} ${this.color} ${this.name}"
+    }
+
```

Comment:
Do we need this extension function in the Fragment too, or just the presenter?

Reply:
there isn't a presenter in this design pattern. I could probably just reference it from MVPs presenter. But didn't want to muddy the patterns too much

Reply:
Oh oh oh, that's my bad, didn't notice this was in the MVCFragment. Thought it was in the MVPFragment. Carry on :-)